src/main/java/uk/gov/hmcts/divorce/divorcecase/model/CaseData.java

```diff
@@ -38,6 +38,7 @@
 import static uk.gov.hmcts.ccd.sdk.type.FieldType.FixedRadioList;
 import static uk.gov.hmcts.ccd.sdk.type.FieldType.TextArea;
 
+@SuppressWarnings("PMD")
```

Comment:
do we need this suppression ?

Reply:
Good spot. I added it when I was having trouble with the ccd import and a static constant in this file. But I changed how the suppression method works. I'll remove it